Document/0x06b-Basic-Security-Testing.md

```diff
@@ -176,13 +163,60 @@ $ docker run -it -p 8000:8000 opensecurity/mobile-security-framework-mobsf:lates
 
 #### Accessing the Device Shell
 
-##### On-device Shell App
+One of the most common things you do when testing an app is accessing the device shell. In this section we'll see how to access the iOS shell both remotely from your host computer with/without a USB cable and locally from the device itself.
 
 ##### Remote Shell
 
-##### Connecting to a physical iOS device
+In contrast to Android where you can easily access the device shell using the adb tool, on iOS you only have the option to access the remote shell via SSH. This means also that your iOS device must be jailbroken in order to connect to its shell from your host computer. For this section we assume that you've properly jailbroken your device and have either Cydia (see screenshot above) or Sileo installed as introduced in "Getting Privileged Access". In the rest of the guide we will reference to Cydia, but the same packages should be available in Sileo.
+
+<img src="Images/Chapters/0x06b/cydia.png" alt="iOS App Folder Structure" width="250">
+
+In order to enable SSH access to your iOS device you can install the OpenSSH package. Once installed, be sure to connect both devices to the same Wi-Fi network and take a note of the device IP address, which you can find in the Settings -> Wi-Fi menu and tapping once on the info icon of the network you're connected to.
+
+You can access now the remote device's shell by running `ssh root@<device_ip_address>` and you'll already have root access, for example:
+
+```shell
+$ ssh root@192.168.197.234
+root@192.168.197.234's password:
+iPhone:~ root#
+```
+
+> press Control + D or type `exit` to quit
+
+When accessing your iOS device via SSH consider the following:
 
-##### Connecting to an iOS emulator
+- The default users are `root` and `mobile`.
+- The default password is `alpine`.
```

Comment:
The default password is `alpine`. -> The default password for both is `alpine`.

Reply:
done